src/signer/noscrypt_signer.cpp

```diff
@@ -258,21 +275,21 @@ inline void NoscryptSigner::_setRemotePublicKey(const string value)
 
 #pragma region Setup
 
+
 void NoscryptSigner::_initNoscryptContext()
```

Comment:
Should we consider pulling this into the constructor, or maybe making it static and passing pointers? Just semantics, not sure your code style idea, I know some (myself) don't like calling mutators in the constructor.

Reply:
We could make it static and have it return the noscrypt context struct, then the constructor can be responsible for setting the context in the signer's internal state.